src/merge.py

```diff
@@ -0,0 +1,52 @@
+"""Merge sort module."""
+
+
+def merge_sort(seq):
+    """Sort sequence by merge sort."""
+    if isinstance(seq, list):
+        mid = len(seq) // 2
+        left, right = seq[:mid], seq[mid:]
+        if len(left) > 1:
+            left = merge_sort(left)
+        if len(right) > 1:
+            right = merge_sort(right)
+        sorted_seq = []
+        while left and right:
+            if left[-1] >= right[-1]:
+                sorted_seq.insert(0, left.pop())
+            else:
+                sorted_seq.insert(0, right.pop())
```

Comment:
This is a little odd to compare the last values and pop off from the end of the list. One of the issues with inserting into a list is that for a list of size `N` inserting at the front is an O(N) operation